client/src/App.tsx

```diff
@@ -36,6 +41,30 @@ function App() {
     });
   }, []);
 
+  /** Opens the snackbar with the provided message */
+
+  function showNotification(message: any, severity: any) {
+    setIsSnackbarOpen(true);
+    setSnackbarMessage(message);
+    setAlertColor(severity);
+  }
+
+  function handleCloseSnackbar(event: any, reason: any) {
```

Comment:
According to the docs, the types should be

`event?: React.SyntheticEvent, reason?: string`